core-java-modules/core-java-io-5/src/test/java/com/baeldung/writestr/WriteStringToOutputStreamUnitTest.java

```diff
@@ -0,0 +1,73 @@
+package com.baeldung.writestr;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+
+import org.junit.jupiter.api.*;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+class WriteStringToOutputStreamUnitTest {
+
+    private static final String UTF8_STRING = "Hello";
+
+    private static final byte[] UTF8_BYTES = UTF8_STRING.getBytes(StandardCharsets.UTF_8);
+
+    private static ByteArrayOutputStream outputStream;
+
+    @BeforeEach
+    void setup() {
+        outputStream = new ByteArrayOutputStream();
+    }
+
+    @Test
+    void whenWriteEncodedBytesToOutputStream_thenBytesAreEqual() throws IOException {
+
```

Comment:
Let's remove the empty first lines of each of the methods, as per the guideline (no blank line after opening curly brace or before closing curly brace).

Reply:
Lines removed. Thanks